Gemfile

```diff
@@ -33,37 +32,35 @@ gem "ransack"
 gem "stackprof"
 gem "sentry-ruby"
 gem "sentry-rails"
-gem "rails-settings-cached"
-gem "octokit"
 
 # Active Storage
 gem "aws-sdk-s3", require: false
-gem "azure-storage-blob", "~> 2.0", require: false
```

Comment:
After a closer look at this—was causing Faraday to be forced to an earlier version and the maintainers have issued a deprecation notice (still in Rails guides though, which was why it was introduced)